### PR TITLE
Update trc.py

### PR DESCRIPTION
**Pull request by suresync**
Added ufw install (should be added to your other scripts as well) and software-properties-common and some extra dependencies for the terracoin client.

### CODE REVIEW

File: trc.py [hunk 1]
```diff
@@ -98,6 +98,7 @@ def chech_root():
 
 def secure_server():
     print_info("Securing server...")
+    run_command("apt-get install ufw")
```
Comment (u3mur4) [inline trc.py:101]: This package is installed by default on ubuntu 16.04. Any reason adding it explicitly?

File: trc.py [hunk 2]
```diff
@@ -118,11 +119,15 @@ def setup_wallet():
         f.close()
 
     print_info("Installing wallet dependencies...")
+    run_command("apt-get install software-properties-common -y")
```
Comment (u3mur4) [inline trc.py:122]: Opps, I incidental put it a few lines below.

File: trc.py [hunk 3]
```diff
@@ -118,11 +119,15 @@ def setup_wallet():
...
     run_command("add-apt-repository ppa:bitcoin/bitcoin -y")
     run_command("apt-get update")
     run_command("apt-get --assume-yes install git unzip libboost-system-dev libboost-filesystem-dev libboost-chrono-dev "
                 "libboost-program-options-dev libboost-test-dev libboost-thread-dev software-properties-common libdb4.8-dev "
-                "libdb4.8++-dev libminiupnpc-dev libevent-dev libzmq3-dev")
+                "libdb4.8++-dev libminiupnpc-dev libevent-dev libzmq3-dev libboost-chrono1.58.0 libboost-filesystem1.58.0 "
+                "libboost-system1.58.0 libboost-thread1.58.0 libdb4.8++ libevent-2.0-5 libevent-core-2.0-5 libevent-pthreads-2.0-5 "
+                "libminiupnpc10 libsodium18 libboost-chrono1.58.0 libboost-filesystem1.58.0 libboost-system1.58.0 "
+                "libboost-thread1.58.0 libdb4.8++ libevent-2.0-5 libevent-core-2.0-5 libevent-pthreads-2.0-5 libminiupnpc10 libsodium18 libzmq5")
```
Comment (u3mur4) [inline trc.py:131]: Thank for adding the extra libraries. I will delete the duplicated libraries.

File: trc.py (hunk 1 above)
Comment (dutchcoin) [reply trc.py:101]: It is missing in multiple default Ubuntu 16.04 vps providers (tested 3 andere none had ufw)

File: trc.py (hunk 3 above)
Comment (dutchcoin) [reply trc.py:131]: This should solve the open issue.

**Review by u3mur4 — APPROVED** [review]

**Comment by u3mur4** [comment]
Thank you for your cooperation!

**Pull request merged**